### ISSUE TITLE
Backlight turns off on 2DS

### ISSUE THREAD

**Issue by Robomandude** [Core-2-Extreme/Video_player_for_3DS#28]
When playing a video in full screen, the bottom backlight turns off.  This is fine on all models except for the 2DS which only has one backlight.  Doing this on the 2ds just makes the screen completely black except under very bright light.
A simple fix is to detect 2ds hardware and not turn off the backlight in such cases

**Comment by Core-2-Extreme** [comment]
Is it for only old2DS, right?
I don't have any 2DS so could you test if it is actually fixed?
[Video_player_for_3DS.zip](https://github.com/Core-2-Extreme/Video_player_for_3DS/files/7283176/Video_player_for_3DS.zip)

**Comment by Robomandude** [comment]
Yeah its only for old2DS, its the only model that uses one hardware screen to emulate 2 separate screens.

It fixes the issue of the backlight turning off, but the bottom screen also stays bright as normal.
If you want similar operation between 2ds and other models the bottom screen will need to be manually blacked out I guess
Here is some quick before and after videos

https://user-images.githubusercontent.com/14086052/136104766-785cf324-e52d-4214-a85e-91d8a6007088.mp4

https://user-images.githubusercontent.com/14086052/136104785-d1bbecee-9a93-4ce1-ba13-c77fbd6f5f3c.mp4
